data_test.go

```diff
@@ -49,6 +49,11 @@ func TestToIngressifyRule(t *testing.T) {
 			t.Errorf("Missing rule: %s", missingRule)
 		}
 	}
+	for _, r := range ingressifyRules {
+		if r.Hash == 0 {
+			t.Errorf("Invalid hash for Ingressify Rule. Hash = %d", r.Hash)
```

Comment:
Just a suggestion, pls feel free to ignore :) But can we test the hash extensively here, and ignore testing it in other places :) i.e. Here we test if the hash really changes if any of the input values change (i.e. if any of namespace/name/service/host/port changes, hash should also change). This can be excessive, hence optional and feel free to ignore :)

Reply:
I think i would rather skip the hash test. if I want this test to be blackbox style then I would need to test that all possible combinations of variations of `(name,namespace,host,path)` makes the hash to change and at that point it feels I'm testing the hash library itself. I think ideally we would want to be able to test that `name, namespace,... == unhash(hash(namespace,host,path))` , but of course, this is not possible